src/resolver/Mutation/remove.js

```diff
@@ -1,6 +1,6 @@
 export default (entityData = []) => (_, { id }) => {
-    const parsedId = parseInt(id, 10); // FIXME fails for non-integer ids
-    const indexOfEntity = entityData.findIndex(e => e.id === parsedId);
+    const stringId = `${id}`;
```

Comment:
Why not calling `toString` instead of using string literals?

Reply:
Mostly habit, because it's shorter.  It has a bug, though: it treats an undefined is the same as the literal string "undefined".  I'll fix that and update.